client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -165,6 +166,18 @@
         this.edit = edit;
         this.editItem = null;
 
+        this.isDeskChanged = function (){
+            return desks.changeDesk;
+        }
+
+        this.highlightsDeskChanged = function (){
+            if(desks.changeDesk) {
+                $location.url('/workspace/monitoring');
+            }
+
+            return true;
```

Comment:
Is this needed? The method always returns the same (and if location is changed, it doesn't really matter anymore).